tests/dummy/app/pods/components/frost-create-form-with-scroll/template.hbs

```diff
@@ -1,18 +1,22 @@
-{{#frost-modal-input title='Test title' subtitle='Subtitle'}}
-{{!#frost-modal-input titleComponent='myTitle'}}
-
-  {{#perfect-scroll
+{{#frost-modal-input
+  title='Test title'
+  subtitle='Subtitle'
+  openButton='Open Modal with Perfect Scroll'
+  openButtonClasses='medium secondary'
+  modalName='my-ps-modal'}}
+  {{#block-slot slot 'form'}}
+    {{#perfect-scroll
```

Comment:
Can we move `perfect-scroll` inside of the form block-slot in the component template?